src/migrate.rs

```diff
@@ -28,21 +28,25 @@ use serde::Serialize;
 pub fn migrate(
     deps: DepsMut<ProvenanceQuery>,
     _: Env,
-    _: MigrateMsg,
+    migrate_msg: MigrateMsg,
 ) -> Result<Response<ProvenanceMsg>, ContractError> {
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
 
     let old_state: StateV1_0_0 = singleton_read(deps.storage, CONFIG_KEY).load()?;
```

Comment:
oof I just realized this is going to be no Bueno, the migrations from V1_0_0 are already preformed, this read would 💥